docs/scos/dev/feature-integration-guides/202200.0/multiple-abstract-products-promotions-and-discounts-feature-integration.md

```diff
@@ -0,0 +1,190 @@
+
+{% endinfo_block %}
+
+### 3) Add translations
+
+Append glossary according to your configuration:
+
+**data/import/common/common/glossary.csv**
+
+```yaml
+cart.title.available_discounts,Verfügbare Rabatte,de_DE
+cart.title.available_discounts,Available discounts,en_US
+```
+
+Run the following console command to import data:
+
+```bash
+console data:import glossary
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that in the database the configured data are added to the `spy_glossary` table.
+
+{% endinfo_block %}
+
+### 4) Add Zed translations
+
+Generate a new translation cache for Zed:
+
+```bash
+console translator:generate-cache
+```
+
+{% info_block warningBox "Verification" %}
+
+Make sure that all labels and help tooltips in the discount form has English and German translation.
+
+{% endinfo_block %}
+
+### 5) Set up behavior
+
+Set up the following behaviors:
+
+| PLUGIN                                                      | SPECIFICATION                                                        | PREREQUISITES | NAMESPACE                                                                                                            |
+|-------------------------------------------------------------|----------------------------------------------------------------------|---------------|----------------------------------------------------------------------------------------------------------------------|
+| DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin | Adds discount promotion form name postfix to the Add To Cart form.   | None          | SprykerShop\Yves\DiscountPromotionWidget\Plugin\CartPage\DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin |
+
+**src/Pyz/Yves/CartPage/CartPageDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Yves\CartPage;
+
+use SprykerShop\Yves\CartPage\CartPageDependencyProvider as SprykerCartPageDependencyProvider;
+use SprykerShop\Yves\DiscountPromotionWidget\Plugin\CartPage\DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin;
+
+class CartPageDependencyProvider extends SprykerCartPageDependencyProvider
+{
+    /**
+     * @return array<\SprykerShop\Yves\CartPageExtension\Dependency\Plugin\AddToCartFormWidgetParameterExpanderPluginInterface>
+     */
+    protected function getAddToCartFormWidgetParameterExpanderPlugins(): array
+    {
+        return [
+            new DiscountPromotionAddToCartFormWidgetParameterExpanderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the plugin works correctly:
+
+1. [Create a discount](/docs/scos/user/back-office-user-guides/{{page.version}}/merchandising/discount/creating-cart-rules.html) and select **Promotional Product** radio button in **Discount Calculation** form.
+2. Add a product abstract sku into the **Abstract Product SKU(s)** field.
+3. Create another discount with the at least one same **Promotional Product**.
```

Comment:
Is this required? Why are we asking them to create a discount with at least one of the same products as the other discount?

Reply:
@JacquieRooney, the plugin allows displaying the same product in several discount carousels on the cart page.

Reply:
Understood, thanks for clarifying.